src/Microsoft.DotNet.Darc/src/DarcLib/Models/AzureDevOps/AzureDevOpsBuild.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.DotNet.DarcLib
+{
+    public class AzureDevOpsBuild
+    {
+        public long Id { get; set; }
+
+        public string BuildNumber { get; set; }
+
+        public AzureDevOpsBuildDefinition Definition { get; set; }
```

Comment:
Why not just call this an "IdNamePair" and remove the empty types? Is there more information there?

Reply:
I created this type to make more clear what the property was referring to. No functionality thing involved.

Reply:
The empty type makes it a bit harder to know though... since now I've got to go look at that other type, find it empty, then look up the other type to figure out it's really just a key/value pair. But it's mostly an opinion for me.  Might be an innocent bystander of the every class in it's own file that makes it a bit... circuitous.